### DISCUSSION TITLE
Welcome to open-source-test Discussions!

### DISCUSSION THREAD

**Discussion by bhavyakrishna001** [bhavyakrishna001/open-source-test#1] [Q&A] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by karthik558** [comment, marked as answer]
scanf("Hello World")/n;
scanf("Hello nub girl 😂😂");
loop ();

**Reply by bhavyakrishna001** [reply]
Fool
